src/modules/users/SetPasswordJob.cpp

```diff
@@ -90,7 +92,14 @@ SetPasswordJob::exec()
         return Calamares::JobResult::ok();
     }
 
-    QString encrypted = QString::fromLatin1( crypt( m_newPassword.toUtf8(), make_salt( 16 ).toUtf8() ) );
```

Comment:
Leave this alone by replacing the implementation of `make_salt()` instead. It could be a `const QString` though.